### DISCUSSION TITLE
Is the code which generated the animation in the Readme public?

### DISCUSSION THREAD

**Discussion by Boxylmer** [MilesCranmer/PySR#458] [Q&A] [answered]
Hi everyone! I was wanting to generate animations for specific problems in my field in a similar style to the (beautiful) animation in the Readme, but I wasn't able to immediately find the code that generated it or similar questions asking for this here or in the issues page. Does anyone know where it was created or if the code is public? I'd love to do something similar, though it looks like it was generated using the 3B1B style library.

**Comment by MilesCranmer** [comment, marked as answer]
Hey @Boxylmer,
The animation source code is in the pysr paper repo here: https://github.com/MilesCranmer/pysr_paper in the `animations` folder. I describe the process in `animations/README.md`.
Cheers,
Miles

**Reply by Boxylmer** [reply]
Hey @MilesCranmer, thanks for the quick response and pointing out the library! I'll look through it and hopefully be able to make something like what you all have!
